src/www/cordova_index.html

```diff
@@ -20,21 +20,8 @@
     <meta charset="UTF-8" />
     <title>Outline</title>
     <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=no" />
-    <link rel="stylesheet" href="style.css" />
-    <!-- See https://github.com/webcomponents/custom-elements#es5-vs-es2015  -->
-    <script src="bower_components/webcomponentsjs/custom-elements-es5-adapter.js "></script>
-    <script src="babel-polyfill.min.js"></script>
-    <script src="bower_components/outline-i18n/index.js"></script>
-    <script src="bower_components/punycode/punycode.min.js"></script>
-    <script src="bower_components/webcomponentsjs/webcomponents-loader.js"></script>
     <script src="cordova.js"></script>
-    <script src="cordova_main.js"></script>
-    <script>
-      window.Polymer = {
-        rootPath: location.pathname.substring(0, location.pathname.lastIndexOf("/") + 1),
-      };
-    </script>
-    <link rel="import" href="ui_components/app-root.html" />
+    <!-- TODO: Set the path we intend to use to serve the app from. -->
```

Comment:
What is the purpose of this setting?

This seems to have changed in Polymer 3. I still need to make these changes and test the affected platforms.

**TODO:** https://polymer-library.polymer-project.org/3.0/docs/devguide/settings

Reply:
The `rootPath` property is used to generate URLs that are relative to the application root. It defaults to "/", so it may not be necessary to set it explicitly. We can now set it in `cordova_main.ts` if needed.

Reply:
Removing it for now then. Let's verify when we test.